src/components/encoding-pane/filter-pane.tsx

```diff
@@ -205,9 +205,14 @@ const filterShelfTarget: DropTargetSpec<FilterPaneProps> = {
     if (monitor.didDrop()) {
       return;
     }
-    const {filter} = monitor.getItem() as DraggedFieldIdentifier;
+    const {filter, filters} = monitor.getItem() as DraggedFieldIdentifier;
```

Comment:
`filters` is already a prop of FilterPane.  Why do you need to get it from the dragged field, which semantically should be a single filter that you add.